main.js

```diff
@@ -1,12 +1,19 @@
 var ListenerModule = (function(){
 
-    var initBookListener = function() {
+    var initBookListener = function(callback) {
```

Comment:
could we add comments to the module to make it clear what's a function, what's a function invocation, returning etc.?